JS/lesson-16/index.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+
+// let admin;
+// let userName = 'Jon';
+// admin = userName;
+// alert(admin);
+
+// let ourPlanet;
+// let userName;
+
+// let x = 20;
+// let y = 58;
+// let z = 42;
+// let rezult = x+y+z;
```

Comment:
Для всех операций должны быть пробелы вокруг
```suggestion
// let result = x + y + z;
```